src/mobile/src/ui/views/wallet/Disable2FA.js

```diff
@@ -16,6 +16,7 @@ import Header from 'ui/components/Header';
 import { height } from 'libs/dimensions';
 import { Styling } from 'ui/theme/general';
 import { leaveNavigationBreadcrumb } from 'libs/bugsnag';
+import i18next from 'shared-modules/libs/i18next.js';
```

Comment:
```suggestion

```
Since `t` is passed as a prop (as the component is wrapped with `withNamespaces`), this import isn't necessary